### DISCUSSION TITLE
newDownloadPlus?

### DISCUSSION THREAD

**Discussion by megamorphg** [xiaoxiaoflood/firefox-scripts#188] [Legacy Development]
The main page shows a screenshot with "newDownloadPlus", I couldn't find this script anywhere in the repository... is this deprecated? Also wondering its functionality, if it is the thing that allows Ctrl + Click open the download pop-up (and just download to default folder otherwise).

**Comment by xiaoxiaoflood** [comment]
Yes, that script is not mine and I never had motivation to fix it when it stopped working in... 2018?

You can see from the screenshot that file name is an input text, allowing to rename directly from there. This was one of the features. Most I can't remember. You could also copy source url by doubleclicking it. "*Save and open*" was also a feature, I guess still today Firefox doesn't allow to open `.exe` files directly on Windows, just to save.

**Reply by megamorphg** [reply]
Dang, easily renaming before saving is pretty dang cool (optionally per click).
